yt/fields/field_info_container.py

```diff
@@ -427,14 +427,25 @@ def alias(self, alias_name, original_name, units=None):
             else:
                 units = self[original_name].units
         self.field_aliases[alias_name] = original_name
+        function = TranslationFunc(original_name)
+        if deprecate:
+            function = DeprecatedFunc(original_name, function)
         self.add_field(
             alias_name,
-            function=TranslationFunc(original_name),
+            function=function,
             sampling_type=self[original_name].sampling_type,
             display_name=self[original_name].display_name,
             units=units,
         )
 
+    def add_deprecated_field(self, name, function, sampling_type, **kwargs):
```

Comment:
So my understanding is that we should use `alias` with `deprecate=True` to deprecate an alias, and call `add_deprecated_field` for deprecated fields that are not aliases. Am I right ? (I think you said you were going to add docs, so I'm writing the question I'd like such doc to answer :) )
Why isn't `add_deprecated_field` used in `alias` I don't fully understand though

Reply:
Fixed both of these points.